### PR TITLE
[Merged by Bors] - feat: port Algebra.Ring.AddAut

### PR DESCRIPTION
**Pull request by winston-h-zhang**

### CODE REVIEW

File: Mathlib/Algebra/Ring/AddAut.lean [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  DistribMulAction.toAddAut _ _
+#align add_aut.mul_left AddAut.mulLeft
+
+/-- Right multiplication by a unit of a semiring as an additive automorphism. -/
+def mulRight (u : Rˣ) : AddAut R :=
+  DistribMulAction.toAddAut Rᵐᵒᵖˣ R (Units.opEquiv.symm <| MulOpposite.op u)
+#align add_aut.mul_right AddAut.mulRight
+
+@[simp]
+theorem mul_right_apply (u : Rˣ) (x : R) : mulRight u x = x * u :=
```
Comment (dupuisf) [inline Mathlib/Algebra/Ring/AddAut.lean:41]:
```suggestion
theorem mulRight_apply (u : Rˣ) (x : R) : mulRight u x = x * u :=
```

File: Mathlib/Algebra/Ring/AddAut.lean [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  rfl
+#align add_aut.mul_right_apply AddAut.mul_right_apply
+
+@[simp]
+theorem mul_right_symm_apply (u : Rˣ) (x : R) : (mulRight u).symm x = x * u⁻¹ :=
```
Comment (dupuisf) [inline Mathlib/Algebra/Ring/AddAut.lean:46]:
```suggestion
theorem mulRight_symm_apply (u : Rˣ) (x : R) : (mulRight u).symm x = x * u⁻¹ :=
```

**Review by dupuisf — COMMENTED** [review]
Just some minor naming changes, but otherwise looks good!

bors d+

**Comment by winston-h-zhang** [comment]
bors +r

**Pull request closed without merging**